### PR TITLE
Added Near Protocol to examples

### PR DESCRIPTION
**Pull request by amirsaran3**
This PR will add Near Testnet to list of chains in example.

### CODE REVIEW

**Comment by bkrem** [comment]
Hi @amirsaran3,

Thank you for taking the time to add NEAR support for our Sign v2 example dapp and wallet 🙌 

It's looking very promising overall, but the wallet deployment still seems to fail on a type error (log output attached below).

Once we can compare the wallet and dapp deployments side by side we'll do a proper review 👍 

```
Failed to compile.
--
14:56:55.102 |  
14:56:55.102 | ./src/components/RequestDetalilsCard.tsx:32:53
14:56:55.103 | Type error: Property 'name' does not exist on type 'never'.
14:56:55.103 |  
14:56:55.103 | 30 \|                   COSMOS_MAINNET_CHAINS[chain as TCosmosChain]?.name ??
14:56:55.103 | 31 \|                   SOLANA_CHAINS[chain as TSolanaChain]?.name ??
14:56:55.103 | > 32 \|                   NEAR_CHAINS[chain as TNearChain]?.name ??
14:56:55.103 | \|                                                     ^
14:56:55.103 | 33 \|                   chain
14:56:55.103 | 34 \|               )
14:56:55.104 | 35 \|               .join(', ')}
```

**Comment by amirsaran3** [comment]
Hey @bkrem, thanks for the review. I have fixed the build issue.

File: dapps/react-dapp-v2/src/contexts/JsonRpcContext.tsx [hunk 1]
```diff
@@ -690,6 +695,95 @@ export function JsonRpcContextProvider({
...
+              }
+            },
+          },
+        });
+
+        return {
+          method,
+          address,
+          valid: true,
+          result: JSON.stringify(result),
```
Comment (bkrem) [inline dapps/react-dapp-v2/src/contexts/JsonRpcContext.tsx:732]: Output is too large for the modal, condensing to only display the `transaction` key:

```suggestion
          result: JSON.stringify((result as any).transaction),
```

File: dapps/react-dapp-v2/src/contexts/JsonRpcContext.tsx [hunk 2]
```diff
@@ -690,6 +695,95 @@ export function JsonRpcContextProvider({
...
+              ],
+            }
+          },
+        });
+
+        return {
+          method,
+          address,
+          valid: true,
+          result: JSON.stringify(result),
```
Comment (bkrem) [inline dapps/react-dapp-v2/src/contexts/JsonRpcContext.tsx:781]: Output is too large for the modal, condensing to only display the transaction keys:

```suggestion
          result: JSON.stringify((result as any).map((r: any) => r.transaction)),
```

**Review by bkrem — APPROVED** [review]

**Comment by bkrem** [comment]
Great work on this, thank you again for taking the time to add NEAR! 💯

**Pull request merged**